### PR TITLE
Added the metadata section

### PR DESCRIPTION
**Pull request by AVRahul**
Added the metadata component section .

Screenshot from 2019-04-01 11-24-06

Screenshot from 2019-04-01 14-49-52

< Signed-off-by: A V RAHUL a.v.rahul11@gmail.com >

### CODE REVIEW

File: docs/products/headers.mdx [hunk 1]
```diff
@@ -146,3 +146,68 @@ import { Playground, PropsTable } from 'docz'
...
 </Playground>
+
+### Example seven
+##### Metadata bar
+
+
+<Playground>
+ <div class="outlet-header-bar">
+    <div class="outlet-header-meta-bar">
+      <div class="outlet-header-meta-bar_title">
```
Comment (visualapps) [inline docs/products/headers.mdx]: .outlet-header-meta-bar_item

File: docs/products/headers.mdx [hunk 2]
```diff
@@ -146,3 +146,68 @@ import { Playground, PropsTable } from 'docz'
...
+        <span>Schedule</span>
+      </div>
+      <div class="outlet-header__meta-bar-text">
```
Comment (visualapps) [inline docs/products/headers.mdx]: outlet-header__meta-bar-value

File: scss/core/abstracts/_variables.scss [hunk 3]
```diff
@@ -774,6 +774,12 @@ $thumbnail-box-shadow:              0 1px 2px rgba($black, .075) !default;
 $figure-caption-font-size:          90% !default;
 $figure-caption-color:              $gray-600 !default;
 
+// Metadata
```
Comment (visualapps) [inline scss/core/abstracts/_variables.scss]: put this variable in the page header section. Line no 223

File: scss/product/components/_headers.scss [hunk 4]
```diff
@@ -47,12 +52,29 @@
   }
 }
 
+.outlet-header-meta-bar {
+  display: grid;
+  flex-direction: column;
+  justify-content: center;
+  border-left: 1px solid $metadata-line-color;
+  text-align: justify;
+  text-justify: inter-word;
```
Comment (visualapps) [inline scss/product/components/_headers.scss]: text-justify is showing `unknown property in chrome`

**Comment by visualapps** [comment]
mention one more example which includes page header, content-header, content (table) and meta-bar

File: scss/product/components/_headers.scss (hunk 4 above)
Comment (AVRahul) [reply scss/product/components/_headers.scss]: chrome does not support `inter-word` property

Comment (AVRahul) [reply scss/product/components/_headers.scss]: Hence, removed the property to maintain consistency.

**Pull request merged**